AdobeStockImageAdminUi/Test/Mftf/Suite/adobe-stock-integration-configuration.xml

```diff
@@ -16,6 +16,10 @@
             </actionGroup>
         </before>
         <after>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminAdobeStockDisableActionGroup" stepKey="disableAdobeStock"/>
```

Comment:
Why do we need to disable Adobe Stock after the test? It's enabled by default and needs to be enable for most tests?